Makefile

```
@@ -309,8 +309,16 @@ check-copyright:
 	@echo "check-copyright"
 	@./scripts/check-copyright.sh
 
+# LINT_NEW_FROM_REV: when set (e.g. to a base commit SHA), only checks lines
+# that changed since that revision. Used in CI to avoid flagging pre-existing
+# issues. If unset, checks all files.
+# Directory exclusions (e.g. tests/) are configured in .golangci.yml.
 check-static: tools/bin/golangci-lint
```

checkmake (0.3.2)

[warning] Target "check-static" should be declared PHONY.

(phonydeclared)